src/platform/README.md

```diff
@@ -0,0 +1,43 @@
+About this directory
+====================
+
+# mnt-sd/
+This directory is a recipe of the initial script.
+It would auto mount partition 1 of the SD card to /mnt and source init.sh script in /mnt if it exists.
+To use this recipe, copy it to a yocto or petalinux project. If you still don't know how to use it, please google yocto/petalinux and learn.
+
+# recipes-xrt/
+This directory including all of the necessary recipes to enable XRT on an embedded system.
+Copy the whole directory to your yocto or petalinux project. You could reference src/runtime\_src/tools/scripts/peta\_build.sh to learn how to use it.
```

Comment:
All these `\` in this file does not seem the correct way to typeset these paths and other symbols...

Reply:
Use backtick instead. Same for the other `.md` files.

Reply:
@keryell I didn't aware what should be the best way to typeset all paths. Use backtick would be a good idea. Changed in #1548.